<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addclient</class>
 <widget class="QDialog" name="addclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Cliente</string>
  </property>
  <widget class="QLabel" name="lbl_RG">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>RG</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Cidade">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_CPF">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Endereco">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Endereco</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Nome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_feedback">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Insira os dados do cliente e clique em Cadastrar.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cadastrar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Nome">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Endereco">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Cidade">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_RG">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_CPF">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Sair">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Emoticon">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_Nome</tabstop>
  <tabstop>line_Endereco</tabstop>
  <tabstop>line_Cidade</tabstop>
  <tabstop>line_RG</tabstop>
  <tabstop>line_CPF</tabstop>
  <tabstop>btn_cadastrar</tabstop>
  <tabstop>btn_Sair</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
